src/app/content/search/utils.ts

```diff
@@ -171,3 +166,15 @@ export const highlightResults = (
     return {result: hit, highlights: hitHighlights};
   })
   ;
+
+export const findSearchResultHit = (
+  results: SearchResultHit[],
+  target: SearchScrollTarget
+): SearchResultHit | undefined => {
+  return results.find((result) => result.source.elementId === target.elementId);
+};
+
+export const isSearchScrollTarget = (target: ScrollTarget): target is SearchScrollTarget => {
+  if (target.type === 'search' && typeof target.index === 'number') { return true; }
+  return false;
+};
```

Comment:
Similar comment to the other guard here. `search/guards.ts` exists already in this case